### PR TITLE
Stop using gorilla/context

### PR DESCRIPTION
**Pull request by sethvargo**
It's unmaintained and leaks memory

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -9,45 +9,34 @@ cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxK
 cloud.google.com/go v0.45.1/go.mod h1:RpBamKRgapWJb87xiFSdk4g1CME7QZg3uwTez+TSTjc=
```
Comment (sethvargo) [inline go.sum:9]: I'm not gonna pretend to understand why removing context created such a large diff on the sum

File: pkg/controller/user/deleteuser.go [hunk 2]
```diff
@@ -43,22 +41,12 @@ func NewDeleteController(ctx context.Context, config *config.ServerConfig, db *d
 func (udc *userDeleteController) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer http.Redirect(w, r, "/users", http.StatusSeeOther)
 
-	user, err := controller.GetUser(w, r)
```
Comment (sethvargo) [inline pkg/controller/user/deleteuser.go:46]: This was never used

File: pkg/controller/user/saveuser.go [hunk 3]
```diff
@@ -47,38 +46,26 @@ func NewSaveController(ctx context.Context, config *config.ServerConfig, db *dat
 }
 
 func (usc *userSaveController) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	defer http.Redirect(w, r, "/users", http.StatusSeeOther)
-
-	user, err := controller.GetUser(w, r)
```
Comment (sethvargo) [inline pkg/controller/user/saveuser.go:52]: This was never used

File: pkg/controller/user/saveuser.go [hunk 4]
```diff
@@ -47,38 +46,26 @@ func NewSaveController(ctx context.Context, config *config.ServerConfig, db *dat
...
-	usc.logger.Infof("FLASH: %+v", flash)
-
-	m := html.GetTemplateMap(r)
-	m["user"] = user
-
 	var form formData
 	if err := controller.BindForm(w, r, &form); err != nil {
-		usc.logger.Errorf("error parsing form: %v", err)
 		flash.Error("Failed to process form: %v", err)
-		controller.WriteJSON(w, http.StatusBadRequest, nil)
```
Comment (sethvargo) [inline pkg/controller/user/saveuser.go:68]: Unclear why we were returning JSON?

Comment (mikehelmick) [reply pkg/controller/user/saveuser.go:68]: yeah - redirect is right here, it's a human used page.

**Pull request merged**